### PR TITLE
Add empty session tips for Auto

### PR DESCRIPTION
**Pull request by YoungeunKim**
:Release Notes:
Add empty session tips for Auto

:Detailed Notes:
After flashing booting, user should select "profile icon" on the screen.
At this time, the session list is empty. So, CLI commands print
unexpected results. Add tips message to user

:Testing Performed:
1. Before run TC, you must select a profile on each display.
    TC passed on AUTO & OSE target and Auto emulator.
2. eslint passed
3. Verified with CLI commands
   a. boot Auto board. The login UI is shown
   b. Run CLI commands. "select profile" tips should be printed.
```
$ ./ares-install.js com.domain.app_1.0.0_all.ipk 
$ ./ares-launch.js -r
$ ./ares-launch.js -H sampleWeb
$ ./ares-launch.js -c com.test.app
$ ./ares-device.js -s
$ ./ares-inspect.js -s com.domian.app.test
$ ./ares-install.js -l
$ ./ares-install.js -F
$ ./ares-shell.js -dp 1
$ ./ares-log.js -ul -dp 1 (on Auto target)
> ares-xxxx ERR! [Tips]: Please select the profile on the display
```

:Issues Addressed:
[PLAT-143757] Add "login profile" tips after flash booting

### CODE REVIEW

**Comment by YoungeunKim** [comment]
Code :+1: 
Verified :+1:

File: lib/base/error-handler.js [hunk 1]
```diff
@@ -12,6 +12,9 @@ const log = require('npmlog');
             // service's error message, common
             "for category \"/dev\"" : "Please enable \"Developer options\" on your device",
 
+            // sessionmanager
+            "ENABLE_PROFILE" : "Please select \"Profile icon\" on display",
```
Comment (ssuminahn) [inline lib/base/error-handler.js:16]: how about 
Select a profile on each display.

File: lib/base/error-handler.js [hunk 2]
```diff
@@ -21,6 +24,7 @@ const log = require('npmlog');
             "Cannnot install privileged app on developer mode" : "Please change the app id (app id should not start with 'com.lge', 'com.webos', 'com.palm')",
             "Cannnot remove privileged app on developer mode" : "You cannot remove the privileged app whose app id start with 'com.lge', 'com.webos', 'com.palm'",
             "unable to execute smack" : "Please remove the app and install again",
+            "empty session list, cannot query getAppInfo" : "Please select \"Profile icon\" on display",
```
Comment (ssuminahn) [inline lib/base/error-handler.js:27]: how about 
Select a profile on each display.

File: lib/base/error-handler.js (hunk 1 above)
Comment (ssuminahn) [reply lib/base/error-handler.js:16]: EMPTY_PROFILE?

**Review by ssuminahn — COMMENTED** [review]
1. In case emulator user is developer, "$ ./ares-log.js -ul -dp 1" is displayed "ares-log ERR! [Tips]: Unable to connect to the target device. root access required".
Is this your intention? I think this is okay. I'm asking for confirmation.

2. On Test performed, #1 is needed more explanation. 
For example, Before test run, you must select a profile each display. Pass TC~~

File: lib/base/error-handler.js (hunk 2 above)
Comment (YoungeunKim) [reply lib/base/error-handler.js:27]: I'll change it to "Please select the profile on the display"

File: lib/base/error-handler.js (hunk 1 above)
Comment (YoungeunKim) [reply lib/base/error-handler.js:16]: @ssuminahn , How about "SELECT_PROFILE" ?

**Comment by YoungeunKim** [comment]
> 1. In case emulator user is developer, "$ ./ares-log.js -ul -dp 1" is displayed "ares-log ERR! [Tips]: Unable to connect to the target device. root access required".
>    Is this your intention? I think this is okay. I'm asking for confirmation.
> 2. On Test performed, #1 is needed more explanation.
>    For example, Before test run, you must select a profile each display. Pass TC~~

#1. Yes.  At first, ares-log checks root permission and then sessionlist for -dp 1. So, "Unable to connect to the target device. root access required" is expected Tips
#2. I updated it.

**Comment by YoungeunKim** [comment]
Code 👍 
Verified 👍

**Comment by ssuminahn** [comment]
code :+1: 
verify :+1:

**Review by ssuminahn — APPROVED** [review]

**Pull request merged**